server/src/model.test.ts

```diff
@@ -396,6 +400,106 @@ test("updateActivity does not update properties when passed undefined values", a
   expect(updatedActivity).toEqual(originalActivity);
 });
 
+test("Add and remove promoted content", async () => {
```

Comment:
In these tests, check to see if it results with the expected structure by examining the output of  `LoadPromotedContent()`.